conf/deployment/vsphere/ipi_1az_rhcos_multus_vsan_3m_3w.yaml

```diff
@@ -13,6 +13,13 @@ ENV_DATA:
   compute_memory: '65536'
   fio_storageutilization_min_mbps: 10.0
   is_multus_enabled: true
+  multus_public_net_interface: "ens224"
+  multus_cluster_net_interface: "ens224"
+  multus_create_public_net: true
+  multus_create_cluster_net: true
+  multus_public_net_namespace: "default"
+  multus_cluster_net_namespace: "default"
```

Comment:
@vavuthu I think we need to test some scenarios on `openshift-storage` name space

Reply:
@vavuthu @petr-balogh Most of the customers I’ve seen have configured their NADs in the `openshift-storage` namespace, so I think it’s better to test using the `openshift-storage` namespace instead of `default`.